### DISCUSSION TITLE
One reason for failure to export

### DISCUSSION THREAD

**Discussion by lei-jiang-rug** [marp-team/marp#291] [Q&A] [answered]
Hi,Yuki Hattori,

Greetings. Thank you for your great work "marp for vscode".

It's really cool.

However, I have found a bug.

Failure to export will be reported when I export an MD file to root directory, like "E:\". The problem can be solved when exporting it to any subfolder, like "E:\my_slides". This is very curious. I have spent two days on this problem.

Anyway, thank you for your contribution.

All the best,

Lei Jiang

**Comment by yhatt** [comment]
An original bug marp-team/marp-cli#442 has fixed in the latest release of [Marp CLI v1.7.1](https://github.com/marp-team/marp-cli/releases/tag/v1.7.1) (marp-team/marp-cli#443).

I'm also going to work for update of CLI integrated into the VS Code extension in this weekend. 😎

**Reply by lei-jiang-rug** [reply]
Your work is highly appreciated since it has saved a lot of energy for me. Thank you.

**Comment by yhatt** [comment, marked as answer]
**UPDATE**: Today the latest VS Code extension has released as [v1.5.1](https://github.com/marp-team/marp-vscode/releases/tag/v1.5.1), that is including a patched [Marp CLI v1.7.1](https://github.com/marp-team/marp-cli/releases/tag/v1.7.1) to fix `EPERM` permission error by `mkdir` while outputting.

If you're using VS Code extension, please update the extension to the latest version, and retry to export. 😄

**Comment by lei-jiang-rug** [comment]
font{
        line-height: 1.6;
    }
    ul,ol{
        padding-left: 20px;
        list-style-position: inside;
    }


    
    Hi Yuki,Thank you for telling me the good news. You really did a great job.I've already fallen in love with the extension. Thank you for your great work.All the best, 
    
        
            
        
    
    
         
        
            font{
            line-height: 1.6;
        }font{
            line-height: 1.6;
        }Lei Jiang ***@***.***
        
    
     

    On 4/17/2022 15:04，Yuki ***@***.***> wrote： 




UPDATE: Today the latest VS Code extension has released as v1.5.1, that is including a patched Marp CLI v1.7.1 to fix EPERM permission error by mkdir while outputting.

If you're using VS Code extension, please update the extension to the latest version, and retry to export. 😄
